.github/workflows/tft.yml

```diff
@@ -180,7 +180,7 @@ jobs:
           tf_scope: private
           api_key: ${{ secrets.TF_API_KEY_RH }}
           update_pull_request_status: false
-          tmt_hardware: '{ "memory": ">= ${{ needs.prepare_vars.outputs.memory }} MB" }'
+          tmt_hardware: '{ "memory": ">= ${{ needs.prepare_vars.outputs.memory }} MB", "network": [{"type": "eth", "device-name": "eth0"}, {"type": "eth", "device-name": "eth1"}] }'
```

Comment:
@liangwen12year does this work?

Reply:
We've tested it with Wen in TF and it seems to be working. Will see how TF action likes this syntax.

Reply:
I pair-programmed with @spetrosi on a testing farm machine using the following command, and it works as expected.  

```
ip a
nmcli c
nmcli d
ethtool -P eth0
cat  /sys/class/net/eth0/address
ethtool -P eth1
cat  /sys/class/net/eth1/address
```